Gems/Atom/RPI/Code/Source/RPI.Public/Pass/Pass.cpp

```diff
@@ -1067,6 +1084,23 @@ namespace AZ
             }
         }
 
+        void Pass::RegisterPipelineGlobalConnections()
+        {
+            for (const PipelineGlobalConnection& connection : m_pipelineGlobalConnections)
+            {
+                PassAttachmentBinding* binding = FindAttachmentBinding(connection.m_localBinding);
+                AZ_RPI_PASS_ERROR(binding != nullptr, "Pass::RegisterPipelineGlobalConnections() - Could not find local binding [%s]",
+                                  connection.m_localBinding.GetCStr());
+
+                if (binding)
+                {
+                    m_pipeline->AddPipelineGlobalConnection(connection.m_globalName, binding, this);
```

Comment:
You may want to check m_pipeline pointer in the beginning of this function

Reply:
Will do